main.js

```diff
@@ -36,12 +36,26 @@ function downloadPng() {
 			trackEvent('ui', 'downloadPng');
 		});
 }
+function downloadJPEG() {
+	var node = document.getElementById('diagram')
+	domtoimage.toBlob(document.getElementById('diagram'), {bgcolor: 'white'})
```

Comment:
```suggestion
	domtoimage.toJpeg(document.getElementById('diagram'), { quality: 0.95 })
	    .then(function (dataUrl) {
                var link = document.createElement('a');
                link.download = 'my-image-name.jpeg';
                link.href = dataUrl;
                link.click();
            });
```
https://github.com/tsayen/dom-to-image